src/abstractions/domain/content.ts

```diff
@@ -4,6 +4,7 @@ type HtmlContent = {
   htmlContent: string;
   previewImage: string;
   type: "html";
+  design: undefined;
```

Comment:
I am not sure if it is fine.

`HtmlContent` objects do not have a `design` property. 

If you added this property because of a compilation error, I think that there should be another solution in the place where the error appears.

Reply:
I will change the if inline for a If with braces and wrap the modal.

Reply:
This was resolved using this validation https://github.com/FromDoppler/doppler-editors-webapp/blob/6cdfe33ab45272a03fc97b7a36b733491c3767a2/src/components/Campaign.tsx#L97